builder/dispatchers.go

```diff
@@ -274,6 +315,14 @@ func run(b *Builder, args []string, attributes map[string]bool, original string)
 		return nil
 	}
 
+	// set Cmd manually, this is special case only for Dockerfiles
+	b.Config.Cmd = config.Cmd
+	// set build-time environment for 'run'.
+	b.Config.Env = append(b.Config.Env, cmdBuildEnv...)
+
+	logrus.Debugf("[BUILDER] Command to be executed: %v", b.Config.Cmd)
+	logrus.Debugf("[BUILDER] Environment (applied in order): %v", b.Config.Env)
```

Comment:
If we're concerned about security of the build-time envs then we probably don't want to log them in the clear.  Maybe move this Debugf to be before we add cmdBuildEnv and then if cmdBuildEnv > 0 say " plus X more" or something?